tamperMonkey/CTC-Departures-Section-Auto-Completer.user.js

```diff
@@ -5,10 +5,10 @@
 // @description  Script to automatically fill out CTC sections
 // @author       Reece-Carruthers
```

Comment:
Bump the @version number up 1 number, just makes it easier for people to update using tamper monkey